docs/book/src/clusterctl/developers.md

```diff
@@ -120,8 +120,20 @@ providers:
     type: InfrastructureProvider
 ```
 
+<aside class="note warning">
```

Comment:
Given that now there is a tab for v0.8, the content of this warning can be moved there

Reply:
done